releases/7.6.0.md

```diff
@@ -15,8 +15,6 @@ Arches 7.6.0 Release Notes
 
 - Adds Github actions to build applications, run tests, and compare coverage between branches for projects.
 
-- Plugins now support the configuration boolean `is_standalone`. Standalone plugins do not appear in the sidebar, and do not display the sidebar or application header.
```

Comment:
It looks like this must have been accidentally removed

Reply:
Actually this is correct -- this was already removed in dev/7.6.x, so this looks like a preparatory change for a merge.